docs/reference/slate/schema.md

```diff
@@ -303,17 +303,34 @@ When supplying your own `normalize` property for a schema rule, it will be calle
 
 Raised when the `object` property of a child node is invalid.
 
-### `'child_required'`
+### `child_required_underflow`
 
 ```js
 {
   index: Number,
+  count: Number,
+  limit: Number,
   node: Node,
   rule: Object,
 }
 ```
 
-Raised when a child node was required but none was found.
+Raised when a child node repeats less than required by a rule's `min` property.
+
+### `child_required_overflow`
+
+```js
+{
+  index: Number,
+  count: Number,
+  limit: Number,
+  node: Node,
+  rule: Object,
```

Comment:
In the case of overflow, do we have access to `child` so that we can include it? Just as a useful extra piece of information for people. (It would simplify the default rule already.)

Reply:
The reason I didn't include it is that I'm not sure which of the overflowing nodes should it be, and that it would break symmetry with `child_min_invalid`, but we could add it.

As for the default rule, perhaps it would be better to just remove all overflowing nodes at once?
```
return node.nodes.slice(index - count + limit, index + 1).forEach(child => {
    editor.removeNodeByKey(child.key)
})
```